client/src/boot/index.js

```diff
@@ -59,14 +59,20 @@ function appBoot() {
   // Apply any injector transformations
   applyTransforms();
 
-  // Force this to the end of the execution queue to ensure it's last.
-  window.setTimeout(() => {
-    Injector.load();
-    routes.start(window.location.pathname);
-
+  Injector.ready(() => {
     // add any possible new reducers that were registered
     const newReducer = combineReducers(Injector.reducer.getAll());
     store.replaceReducer(newReducer);
+
+    routes.start(window.location.pathname);
+    if (window.jQuery) {
+      window.jQuery('body').addClass('js-injector-boot');
+    }
+  });
+
+  // Force this to the end of the execution queue to ensure it's last.
+  window.setTimeout(() => {
+    Injector.load();
   }, 0);
```

Comment:
Since this is now just a single method call, we don't need a function literal. `window.setTimeout(Injector.load, 0)`